cpp/tests/copying/concatenate_tests.cpp

```diff
@@ -194,8 +188,14 @@ TEST_F(StringColumnTest, ConcatenateTooManyColumns)
 
 struct TableTest : public cudf::test::BaseFixture {};
 
+template <typename T>
+using column_wrapper = cudf::test::fixed_width_column_wrapper<T>;
```

Comment:
Did this just move from above? Also, couldn't it be used in some of the lines above (e.g. 78-80)? I'm not suggesting expanding the diff of this PR for unrelated changes, more not sure what the purpose of this change was here. If it's intended to improve the scoping of the alias then perhaps we should save that for a future PR where it's used more consistently throughout the file?